.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```

Comment:
Uuuuuh, what? This forces everyone to use LF-newlines on all platforms, which isn't a good idea IMO.

Reply:
I'm editing a few files but `git diff` on Windows shows me changes on all newlines in a bunch of files. I added `.gitattributes` to make `git diff` useful, but that `eol=lf` part I added is probably overkill. Thanks for pointing that out, I'll remove that part again.